tests/neg/varargsInMethodsT1625/firstParamIsVarArgs.scala

```diff
@@ -0,0 +1,3 @@
+trait T1 {
+  def foo(x: String*, y: String): Int // error: *-parameter must come last
```

Comment:
also - just an FYI, the ": error text here" part of the comment is not parsed in any way (currently)

no need to change or anything